tests/test_campaigns.py

```diff
@@ -24,10 +24,6 @@ def setUp(self):
                 'title_register': 'Election Research Preacceptance Competition',
                 'title_landing': 'Welcome to the Election Research Preacceptance Competition!'
             },
-            'osf-preprints': {
```

Comment:
Remove tests for osf-preprints (the one hosted on localhost:5000), which does not provide useful results.